src/components-examples/material/chips/chips-harness/chips-harness-example.spec.ts

```diff
@@ -0,0 +1,63 @@
+import {TestBed, ComponentFixture, waitForAsync} from '@angular/core/testing';
+import {TestbedHarnessEnvironment} from '@angular/cdk/testing/testbed';
+import {MatChipHarness, MatChipListHarness} from '@angular/material/chips/testing';
+import {HarnessLoader} from '@angular/cdk/testing';
+import {BrowserDynamicTestingModule, platformBrowserDynamicTesting}
+  from '@angular/platform-browser-dynamic/testing';
+import {ChipsHarnessExample} from './chips-harness-example';
+import {NoopAnimationsModule} from '@angular/platform-browser/animations';
+import {MatChipsModule} from '@angular/material/chips';
+
+describe('ChipsHarnessExample', () => {
+  let fixture: ComponentFixture<ChipsHarnessExample>;
+  let loader: HarnessLoader;
+  beforeAll(() => {
+    TestBed.initTestEnvironment(BrowserDynamicTestingModule, platformBrowserDynamicTesting());
+  });
+  beforeEach(
+    waitForAsync(() => {
+      TestBed.configureTestingModule({
+        imports: [MatChipsModule, NoopAnimationsModule],
+        declarations: [ChipsHarnessExample]
+      }).compileComponents();
+      fixture = TestBed.createComponent(ChipsHarnessExample);
+      fixture.detectChanges();
+      loader = TestbedHarnessEnvironment.loader(fixture);
+    })
+  );
+
+  it('should get whether a chip list is disabled', async () => {
+    const chipLists = await loader.getAllHarnesses(MatChipListHarness);
```

Comment:
Since you're only testing the first list, you can use `loader.getHarness` instead. It'll help you avoid repeating `[0]` below. The same goes for the other tests that only look for one instance.